README.md

```diff
@@ -421,3 +421,93 @@ To use the Base Modal component, add the following markup to your Blade template
 - `closeable` - (optional) Adds close button at the top right corner. `true` by default
 - `close-on-click-outside` - (optional) Adds behavior to close when clicking outside the modal. `true` by default
 - `close-on-escape-key` - (optional) Adds behavior to close when pressing the Esc key. `true` by default
+
+
+### Tabs
+
+### Underlined
+
+```blade
+<x-kernl-tabs.underlined
+    :tabs="['Zero', 'First', 'Second']"
+>
+    <x-kernl-tabs.underlined.item :index="0">
+        This is the content Zero
+    </x-kernl-tabs.underlined.item>
+
+    <x-kernl-tabs.underlined.item :index="1">
+        This is the content One
+    </x-kernl-tabs.underlined.item>
+
+    <x-kernl-tabs.underlined.item :index="2">
+        This is the content Two
+    </x-kernl-tabs.underlined.item>
+
+</x-kernl-tabs.underlined>
```

Comment:
Oh and can you update the docs for these now that the API has changed? Thanks!

Reply:
Done! 👍